Don't generate debug information during Rust build till following SPARC
issue is resolved:

https://github.com/rust-lang/rust/issues/62001

--- a/src/bootstrap/bootstrap.py
+++ b/src/bootstrap/bootstrap.py
@@ -645,7 +645,7 @@
             if "LIBRARY_PATH" in env else ""
         # preserve existing RUSTFLAGS
         env.setdefault("RUSTFLAGS", "")
-        env["RUSTFLAGS"] += " -Cdebuginfo=2"
+        env["RUSTFLAGS"] += " -Cdebuginfo=0"
 
         build_section = "target.{}".format(self.build_triple())
         target_features = []
